config/m4/compiler.m4

```diff
@@ -449,6 +449,27 @@ AS_IF([test "x$enable_gcov" = xyes],
                                        [AC_LANG_SOURCE([[int main(int argc, char** argv){return 0;}]])])],
       [:])
 
+#
+# Enable stack usage check
+#
+# To suppress -Wframe-larger-than=8192 for specific functions, use:
+# #pragma GCC diagnostic push
+# #pragma GCC diagnostic ignored "-Wframe-larger-than="
+# void specific_function() {
+#     /* Function code */
+# }
+# #pragma GCC diagnostic pop
```

Comment:
do we really want to advise users to do that?

Reply:
Maybe as a tmp solution? they can also use the config flag I guess... removed it